tools/local-storage/@types/globals.d.ts

```diff
@@ -0,0 +1,9 @@
+declare namespace NodeJS {
```

Comment:
We shouldn't need this since we're targeting DOM here. Add a `"dom"` entry to `"lib"` of `tsconfig.json`